js/ui/text_box/ui.text_editor.base.js

```diff
@@ -16,9 +16,12 @@ import TextEditorButtonCollection from './texteditor_button_collection/index';
 import config from '../../core/config';
 import errors from '../widget/ui.errors';
 import browser from '../../core/utils/browser';
+import { getWindow } from '../../core/utils/window';
 import { Deferred } from '../../core/utils/deferred';
 import LoadIndicator from '../load_indicator';
 
+const window = getWindow();
```

Comment:
What about SSR? Wouldn't it be better to move window obtaining to `_collapseInputContainer` method?